lib/vscode/build/hygiene.js

```diff
@@ -62,7 +74,7 @@ function hygiene(some, linting = true) {
 		const lines = file.__lines;
 
 		for (let i = 0; i < copyrightHeaderLines.length; i++) {
-			if (lines[i] !== copyrightHeaderLines[i]) {
+			if (lines[i] !== copyrightHeaderLines[i] && lines[i] !== coderCopyrightHeaderLines[i]) {
```

Comment:
Sooths VSCode’s aggressive commit linter. This eventually changes in v1.59 to an eslint check instead.